spec/controllers/feedback_controller_spec.rb

```diff
@@ -31,6 +31,41 @@
   #     response.should be_success
   #   end
   # end
+
+  describe "check_request_params" do
+    before :each do
+      @params1 = {
+        'feedback' => 'stuff here'
+      }
+      @params2 = {
+        'random' => 'no feedback'
+      }
+    end
+
+    it 'returns false if params does not have feedback key' do
```

Comment:
Can we combine line 45 and 49 into a more general version

Reply:
will do